samples/Basic/Gtk3/SkiaSharpSample/MainWindow.cs

```diff
@@ -49,7 +50,7 @@ private void OnPaintSurface(object sender, SKPaintSurfaceEventArgs e)
 			};
 			using var font = new SKFont
 			{
-				Size = 24
+				Size = 24 * ScaleFactor
```

Comment:
I scaled the text size so that on screen I get the same physical dimension.

When I set the option to true the canvas is twice bigger. If I don't scale the text will be really small.

Reply:
Maybe this is a different issue in gtk... I think what is supposed to happen after my rewrite of the property, false is not actually supposed to be blurry but small. Then the ignore will auto scale up

Reply:
Are you able to test any other platforms?

Reply:
> Maybe this is a different issue in gtk... I think what is supposed to happen after my rewrite of the property, false is not actually supposed to be blurry but small. Then the ignore will auto scale up

Maybe I should rename the property so that there's no ambiguity. `UseScaleFactor` ?

> Are you able to test any other platforms?

I can test WSL.

Reply:
I mean can you test android/ios/mac/windows? Not using GTK that is.

I think the issue is the property was fixed in the other platforms be be better, and gtk is probably still blurry because I have no idea how gtk works.

I think the correct fix should be when the property is false, everything is small and crisp, and when it is true, everything is big and crisp. Basically, the property just enables the autoscaling to match the UI sizes as opposed to using raw pixels.

If we change the property name then it probably should be in a new PR and do it for all the controls. Right now I think the bug is that gtk never got the property fix to scale with the screen density because I did not know how to do it and no one ever complained.

I don't mind fixing it so that false "breaks" things as long as it matches the rest of the sdks. No one wants a blurry image, so that is never a happy path.

Reply:
Hi @mattleibow, I won't have time to work more on this. I believe the best is to close the PR.